### ISSUE TITLE
Events not working when loading a RTMP stream

### ISSUE THREAD

**Issue by iignatov** [flowplayer/flash#188]
It seems that the FlowPlayer events are not working when playing RTMP streams. I saved the [Basic RTMP Demo](http://flash.flowplayer.org/demos/standalone/plugins/streaming/rtmp.html) page locally and added event handlers for the `onBeforePause` clip event and the `onLoad` player event (without any further modifications to the source). None of the events was fired.

Afterwards I added also the following code:
`<button onclick="alert($f().isLoaded())">Test</button>`

After clicking the button, when the player is loaded and the video is already playing, the alert still shows `false` instead of `true` (as expected).

**Comment by pqrosa** [comment]
I have been trying to get events to work when playing RTMP streams but have had no success. Did you find a solution for this problem? I am trying to calculate the loading time of my live stream to use it as a technical KPI.

Updated:
Well, I think the joke was on me. I was trying to do everything locally and hadn't properly configured the flash security settings. The video playback was fine before setting it though. Then I deployed the page to a test server (tomcat) and it also worked.
